src/target/stm32f1.c

```diff
@@ -820,16 +955,22 @@ static bool stm32f1_option_write_erased(
 	return status == SR_PROG_ERROR;
 }
 
+#define STM32F1_OB_COUNT 8U
+#define AT32F4_OB_COUNT 24U
```

Comment:
All `#define`'s should go at the top of the file after the `#include` block please.

Reply:
Moved up.